### ISSUE TITLE
mpv crashes on play

### ISSUE THREAD

**Issue by rubenhorn** [iamtalhaasghar/yewtube#21]
## Issue
When playing a song/playlist with mpv an error occurs. (vlc and mplayer work)
```
    [<-] seek [->]       [9] volume [0]      [CTRL-C] return
    [DN] SEEK [UP]       [space] pause

    Rick Astley - Never Gonna Give You Up (Official Music Video) [03:33]
C:\Python310\lib\subprocess.py:961: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stderr = io.open(errread, 'rb', bufsize)
Invalid message level 'no:statusline=status'
Error parsing option msg-level (option parameter could not be parsed)
Setting commandline option --msg-level=all=no:statusline=status failed.
Exiting... (Fatal error)
```
This does however not crash the whole program and just "resets" the UI to the previous state (e.g. list search results),

## Your Environment
```
> yt --version
yewtube version    : 2.6.5
yt_dlp version     : 2022.02.04
Python version     : 3.10.2 (tags/v3.10.2:a58ebcc, Jan 17 2022, 14:12:15) [MSC v.1929 64 bit (AMD64)]
Processor          : Intel64 Family 6 Model 78 Stepping 3, GenuineIntel
Machine type       : AMD64
Architecture       : 64bit, WindowsPE
Platform           : Windows-10-10.0.19043-SP0
sys.stdout.enc     : utf-8
default enc        : utf-8
Config dir         : C:\Users\Ruben\AppData\Roaming\mps-youtube
```

**Comment by rachmadaniHaryono** [comment]
mpv version?

for example mine

```shell
> mpv --version
mpv 0.33.1 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
FFmpeg library versions:
   libavutil       56.70.100
   libavcodec      58.134.100
   libavformat     58.76.100
   libswscale      5.9.100
   libavfilter     7.110.100
   libswresample   3.9.100
FFmpeg version: 4.4-6ubuntu5
```
based on this, it should be `--msg-level=all=no,statusline=status` instead of `--msg-level=all=no:statusline=status`

https://man.archlinux.org/man/community/mpv/mpv.1.en

**Comment by rubenhorn** [comment]
Thx for the fix.  
I still get the following error message on playback, but at least it works 😀
```
C:\Python310\lib\subprocess.py:961: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stderr = io.open(errread, 'rb', bufsize)
```

mpv version:
```
> mpv --version
mpv 0.34.0 Copyright © 2000-2021 mpv/MPlayer/mplayer2 projects
 built on Sun Nov  7 20:16:38 +08 2021
FFmpeg library versions:
   libavutil       57.7.100
   libavcodec      59.12.100
   libavformat     59.8.100
   libswscale      6.1.100
   libavfilter     8.16.101
   libswresample   4.0.100
FFmpeg version: git-2021-11-06-1728127e
```

If I have time I'll look into this and try to fix it.
Can you re-open the issue?

**Comment by iamtalhaasghar** [comment]
fixed in #23
